internal/backends/rlang/rlang.go

```diff
@@ -13,33 +14,35 @@ func getImports(imports string) []string {
 }
 
 func createRPkgDir() {
-	if _, err := os.Stat("./R"); os.IsNotExist(err) {
-		if err = os.MkdirAll("./R/x86_64-pc-linux-gnu-library/3.4", os.ModeDir); err != nil {
+	home := os.Getenv("HOME")
+
+	if _, err := os.Stat(path.Join(home, "R")); os.IsNotExist(err) {
+		rPath := path.Join(home, "R", "x86_64-pc-linux-gnu-library", "3.4")
```

Comment:
Some package managers in upm install into a directory under $HOME, some install elsewhere, and some install into the current working directory. I'm not sure what the most friendly choice for R is - under $HOME is fine and dandy. But you don't _have to_ install under $HOME if you don't want to.

Reply:
Yeah, I had just done that because this is where R installs packages if it doesn't have perms to install to the normal directory (located somewhere in /usr/lib I think).